src/edu/csus/ecs/pc2/shadow/RemoteRunSubmitter.java

```diff
@@ -52,6 +52,22 @@ public RemoteRunSubmitter(IInternalController controller) {
      * @param overrideRunId - override run id
      */
     public void submitRun(String clientIdString, String problemID, String languageID, IFile mainFile, List<IFile> auxFiles, long overrideTimeMS, long overrideRunId) {
+        submitRun(clientIdString, problemID, languageID, null, mainFile, auxFiles, overrideTimeMS, overrideRunId);
+    }
+    
+    /**
+     * Submit a run to the server.
+     * 
+     * @param clientIdString - "teamN" client name
+     * @param problemID - problem id
+     * @param languageID - CLICS language id
+     * @param entry_point - java/kotlin entry point, null if default or none
```

Comment:
I don't particularly care for the fact that the above line explicitly refers to "java/kotlin", when according to the CLICS spec it could be non-null for ANY language other than Python, C, or C++.

Not something I'd deny Approval for; just a comment.

Reply:
Agreed.  Changed comment.